Fabric.Identity.API/appsettings.json

```diff
@@ -26,9 +26,9 @@
     "Password": ""
   },
   "HostingOptions": {
-    "UseIis": false,
-    "UseTestUsers": true,
-    "StorageProvider": "InMemory",
+    "UseIis": true,
+    "UseTestUsers": false,
+    "StorageProvider": "sqlserver",
```

Comment:
I'm thinking this file should be undone, but it seems like we are always making these changes to get it to run locally